generators/angular/templates/src/main/webapp/app/admin/logs/logs.component.ts.ejs

```diff
@@ -33,20 +37,35 @@ import { LogsService } from './logs.service';
 export default class LogsComponent implements OnInit {
   loggers?: Log[];
   filteredAndOrderedLoggers?: Log[];
+  isLoading = false;
   filter = '';
   orderProp: keyof Log = 'name';
   ascending = true;
+<%_ if (applicationTypeGateway && serviceDiscoveryAny) { _%>
+  services: string[] = [];
+  selectedService: string | undefined = undefined;
+<%_ } _%>
 
-  constructor(private logsService: LogsService) {}
+  constructor(private logsService: LogsService<%= applicationTypeGateway && serviceDiscoveryAny ? ', private gatewayRoutesService: GatewayRoutesService' : '' %>) {}
```

Comment:
This service has to be declared in the `LogsModule`, but there ist no template file for this. How can I add this provider?